### ISSUE TITLE
Popup should show when changing devices

### ISSUE THREAD

**Issue by kitlei-robert** [irzyxa/Volume2#80]
When activating Set Next Default Device using a keyboard shortcut, the OSD window should show as a visual confirmation of the change of the device. The device is changed, but the OSD currently doesn't pop up.

Note: Show OSD is set to "By time" in Options/OSD; changing the volume using a keyboard shortcut does make the OSD show.

**Comment by ThisIsABug** [comment]
I want to second that request. But for a slightly different cause.

My usecase:
I like to work without the OSD. I use the systray icon only.
There I have the possibility to use the icon for the selected device OR volume.
If the tooltip would pop up on a device change - as it does already while changing the volume - I could have both functions (default device & volume) through one systray icon only.

That would be awesome.

**Comment by irzyxa** [comment]
> When activating Set Next Default Device using a keyboard shortcut, the OSD window should show as a visual confirmation of the change of the device. The device is changed, but the OSD currently doesn't pop up.
> 
> Note: Show OSD is set to "By time" in Options/OSD; changing the volume using a keyboard shortcut does make the OSD show.

Do you have enabled option System - Notifications - Volume? Not all OSD skins contain device name text, so it only shows the current volume. I am not sure it is required.

**Comment by irzyxa** [comment]
Also what Windows version do you use?

**Comment by kitlei-robert** [comment]
> Do you have enabled option System - Notifications - Volume? Not all OSD skins contain device name text, so it only shows the current volume. I am not sure it is required.

You're right, it was turned off. If I turn it on, there is a notification about the changed device. So consider this ticket half closed...

... however, this is somewhat different from the effect that I've described, in the following ways.

1. It adds clutter: needlessly increases the number of notifications.
1. It does not show the current volume beside the new device's name.
1. The notification appears not within the OSD, but at a different corner of the screen.
1. The notification is too verbose; I'd rather see just the new device's name (and the new volume).

> Also what Windows version do you use?

Probably irrelevant, but the newest: 1903, fully updated.

**Comment by DJN0** [comment]
I'd also like to request some kind of OSD notification when I change audio devices. Enabling System-Notification-Volume does make a Windows notification pop up on device change, but the Windows notification is slow and sometimes gets stuck when I'm cycling through multiple playback devices. If the OSD notification for volume change popped up when I changed devices that would be perfect, since the skin I use also indicates the current device.
